specs/testing/skills/SkillsFunctionalTesting.md

```diff
@@ -254,7 +280,7 @@ Given these elements, the number of test cases for each scenario can be calculat
 
 **Testing matrix**
 
-- Skill: TeamsBot
+- Skill: [TeamsBot](https://fuselabs.visualstudio.com/TeamsIntegrationTesting/_git/TeamsIntegrationTesting?path=%2FDotnetIntegrationBot%2FBots%2FIntegrationBot.cs&version=GBmaster&line=94&lineEnd=95&lineStartColumn=1&lineEndColumn=1&lineStyle=plain)
```

Comment:
Same as above (no links to internal repos, add the description to the Skills list section)

Reply:
resolved